packages/core-forger/lib/manager.js

```diff
@@ -145,7 +144,9 @@ module.exports = class ForgerManager {
    * @param {Object} round
    */
   async __forgeNewBlock (delegate, round) {
-      emitter.emit('forger.started', delegate.publicKey)
+      // NOTE: Disabled for now as this could cause a delay in forging that
```

Comment:
I think there should be a TODO here, to work on not missing the blocks and re-enabling the emitter